integreat_compass/core/settings.py

```diff
@@ -122,3 +126,31 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+
+# Media Library
+
+#: URL that handles the media served from :setting:`MEDIA_ROOT` (see :setting:`django:MEDIA_URL`)
+MEDIA_URL = "/media/"
+
+#: Absolute filesystem path to the directory that will hold user-uploaded files (see :setting:`django:MEDIA_ROOT`)
+MEDIA_ROOT = os.environ.get(
+    "INTEGREAT_COMPASS_MEDIA_ROOT", os.path.join(BASE_DIR, "media")
+)
+
+#: The maximum size of media files in bytes
+MEDIA_MAX_UPLOAD_SIZE = int(
+    os.environ.get("INTEGREAT_COMPASS_MEDIA_MAX_UPLOAD_SIZE", 3 * 1024 * 1024)
+)
+
+# Voting system settings
+
+#: Maximum votes cast on new offers
+NEW_OFFER_GREMIUM_SIZE = int(
+    os.environ.get("INTEGREAT_COMPASS_NEW_OFFER_GREMIUM_SIZE", "6")
```

Comment:
```suggestion
    os.environ.get("INTEGREAT_COMPASS_NEW_OFFER_GREMIUM_SIZE", 6)
```
`int()` also works when the number already is an int.